doc/nrf/device_guides/working_with_nrf/nrf70/developing/raw_tx_operation.rst

```diff
@@ -0,0 +1,128 @@
+.. _ug_nrf70_developing_raw_tx_operation:
+
+Raw packet transmission
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nrf70 series device supports the transmission of raw packets. Raw packets are packets that are not touched by the 802.11 MAC layer during transmission. The Application should provide an 802.11 conformant packet with transmit parameters which the 802.11 MAC layer will utilize to transmit packets.
+
+
+.. _ug_nrf70_developing_enabling_raw_packet_transmit_feature:
+
+Enabling raw packet transmit feature
+************************************
+
+The raw packet transmit feature for the nrf70 series device is placed under a configuration flag ``CONFIG_NRF700X_RAW_DATA_TX``. The configuration setting can be enabled by enabling the configuration in the ``prj.conf`` of the application attempting to send raw packets.
+
+.. _ug_nrf70_developing_raw_tx_operating_modes:
+
+Raw packet operating modes
+**************************
+
+The nrf70 series device supports raw packet transmission when TX-Injection mode is enabled. A net_mgmt API ``NET_REQUEST_WIFI_MODE`` has been provided to set TX-Injection mode of operation.
```

Comment:
Probably implicit, but it is worth mentioning that the enabling of the TX Injection mode is a runtime API

Reply:
okay

Reply:
okay